src/components/endpoints/__tests__/Accounts.spec.ts

```diff
@@ -83,7 +83,7 @@ test('can connect to browser extension when authorized', async () => {
 })
 
 test('can disconnect from browser extension', async () => {
-  window.ethereum = {}
+  window.ethereum = {} as any
```

Comment:
Without `any` getting this error:
<img width="1316" alt="Screenshot 2023-02-21 at 14 28 22" src="https://user-images.githubusercontent.com/36865532/220345098-9670274c-cb74-42a1-88a2-6a814ddd34dd.png">